packages/scandipwa/src/component/MyAccountOrderItemsTable/MyAccountOrderItemsTable.style.scss

```diff
@@ -9,6 +9,10 @@
  * @link https://github.com/scandipwa/scandipwa
  */
 
+:root {
+    --my-account-order-items-comment-border-color: #979797;
```

Comment:
Move all values for border inside variable: 
1px solid $color

Same as change name for corresponding one.